lib/resources/majors.tsx

```diff
@@ -2425,144 +2479,238 @@ const bsCS_New: Major = {
         },
       ],
     },
     {
-      name: 'Probability and Statistics Requirements',
-      required_credits: 4,
+      name: 'Mathematics Electives',
+      required_credits: 8,
       min_credits_per_course: 3,
       description:
-        '<p> The BS math courses must include coverage ' +
-        'of both probability and statistics, which can be satisfied in many ways, including ' +
-        'taking any of the 553.3xx combined Probability & Statistics courses. Probability and Statistics:</p><p>Two paths:</p>',
+        '<b>Electives</b> <br /> At least 3 more Math courses must be taken at the 200 or above level, ' +
+        'and must include coverage of both Probability and Statistics.',
       criteria:
-        'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.420[C]^OR^EN.553.430[C]',
+        '(AS Mathematics[D]^OR^EN Applied Mathematics & Statistics[D])^AND^(200[L]^OR^Upper Level[L])',
+      double_count: [
+        'Computer Science',
+        'Humanities/Social Sciences',
+        'Writing Intensive',
+      ],
       pathing: 1,
       fine_requirements: [
         {
           description:
-            '<p>1. Any of the three courses below:</p><p>EN.553.211</p><p>EN.553.310</p><p>EN.553.311</p> ',
+            '<b>Option 1: Probability and Statistics Combined </b> <br />' +
+            'Select one of the following: <br />' +
+            'EN.553.211 Probability and Statistics for the Life Sciences<br />' +
+            'EN.553.310 Probability and Statistics for the Physical Sciences and Engineering<br />' +
+            'EN.553.311 Probability and Statistics for the Biological Sciences and Engineering',
           required_credits: 4,
           criteria: 'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]',
         },
         {
           description:
-            '<p>2. Both Intro to Probability and Intro to Statistics</p><p>EN.553.420</p><p>EN.553.430</p> ',
+            '<b>Option 2: Probability and Statistics Separate </b> <br />' +
+            'EN.553.420 Introduction to Probability<br />' +
+            'EN.553.430 Introduction to Statistics<br />',
           required_credits: 8,
           criteria: 'EN.553.420[C]^OR^EN.553.430[C]',
         },
       ],
     },
     {
-      name: 'Science',
+      name: 'Basic Sciences',
       required_credits: 8,
       min_credits_per_course: 1,
       description:
-        'Students must take two semesters of core science courses (any combination of Physics, ' +
-        'Chemistry, Biology), with their associated labs. AP credit is an acceptable substitute for these courses and labs.',
+        'At least two semesters of physics or two semesters of chemistry, with the associated laboratories, must be included.',
       criteria: 'N[A]',
+      double_count: ['Humanities/Social Sciences', 'Writing Intensive'],
+      exception:
+        'EN Computer Science[D]^OR^AS Center for Language Education[D]^OR^EN Applied Mathematics & Statistics[D]',
```

Comment:
exception does not work. I think Olivia wanted to remove them?

Reply:
Done